ironic-config/ironic.conf.j2

```diff
@@ -216,13 +216,23 @@ ipxe_config_template = /templates/ipxe_config.template
 [redfish]
 use_swift = false
 kernel_append_params = nofb nomodeset vga=normal ipa-insecure=1 {% if env.ENABLE_FIPS_IPA %}fips={{ env.ENABLE_FIPS_IPA|trim }}{% endif %} {% if env.IRONIC_RAMDISK_SSH_KEY %}sshkey="{{ env.IRONIC_RAMDISK_SSH_KEY|trim }}"{% endif %} {{ env.IRONIC_KERNEL_PARAMS|trim }} systemd.journald.forward_to_console=yes
+{% if env.BMC_TLS_ENABLED == "true" %}
+# idrac uses the same options as the redfish driver
+verify_ca = /tmp/bmc-tls.pem
```

Comment:
`/tmp` is not a great place for configuration files, let's use `/conf` instead (provided in ironic-common.sh).

Reply:
Done.